tests/test_schemas.py

```diff
@@ -54,27 +68,42 @@ def load_as_dataset(filepath):
     return iati.Dataset(xml_str)
 
 
-@pytest.mark.parametrize('filepath', list_xml_files_recursively('tests/activity-tests/should-pass/') +  # Legacy tests
-                         list_xml_files_recursively('tests/should-pass/'))  # Tests in new format
-def test_pass_files(activity_schema, filepath):
-    """Check that all 'should-pass' test files are XML and pass Schema validation."""
+@pytest.mark.parametrize('filepath', list_xml_files_recursively('tests/activity-tests/should-pass/') +  # Legacy activity test cases
+                         list_xml_files_recursively('tests/should-pass/iati-activities/') +  # Legacy activity test cases
```

Comment:
Is this coment correct? I thought tests/should-pass were the new test cases for 2.03?

Reply:
Fixed in 684e12b.